readme.md

```diff
@@ -240,7 +240,11 @@ If the file extension is omitted, it is auto-completed to the correct extension
 
 `sign-entitlements` - *String*
 
-  The path to entitlements used in signing. (Currently limited to Mac App Store distribution.)
+  The path to the 'parent' entitlements used in signing. (Currently limited to Mac App Store distribution.)
+  
+ `sign-entitlements-inherit` - *String*
+ 
+  The path to the 'child' entitlements. See [electron-osx-sign](https://www.npmjs.com/package/electron-osx-sign) for more details.
```

Comment:
@malept That may be better I think.
I'll find a way to explain the entitlements for `darwin` builds as they, apart from sandboxing an app, do not actually do much except interacting with iCloud services, iAP or things like that. It's pretty hard to access some of these Apple services with current builds of Electron, so basically entitlements could be ignored for `darwin`. The default entitlements for `darwin` is blank: If a `darwin` build is sandboxed, it should not also really function properly in the presence of frameworks like `Squirrel.framework`.